tensorflow/python/eager/wrap_function.py

```diff
@@ -288,6 +310,10 @@ def _fetch_preprocesing_callback(f):
         operation_fetches + tensor_fetches,
         pruned_graph,
         sources=flat_feeds + internal_captures)
+
+    # Note that we deliberately add the component tensors of any SparseTensors
```

Comment:
Change "SparseTensors" to "composite tensors" here, since it applies to all composite tensors.

Reply:
Will do, thanks!